projects/1.20/assets/netherite-horse-armor-cj/netheritehorsearmor/lang/zh_cn.json

```
@@ -0,0 +1,16 @@
+{
+	"item.netheritehorsearmor.copper_horse_armor": "铜马铠",
+	"item.netheritehorsearmor.chain_horse_armor": "锁链马铠",
+	"item.netheritehorsearmor.emerald_horse_armor": "绿宝石马铠",
+	"item.netheritehorsearmor.netherite_horse_armor": "下界合金马铠",
+	"item.netheritehorsearmor.gilded_netherite_horse_armor": "镀金下界合金马铠",
+	"item.netheritehorsearmor.enderite_horse_armor": "末影合金马铠",
+	"item.netheritehorsearmor.gilded_enderite_horse_armor": "镀金末影合金马铠",
+
+	"tooltip.netheritehorsearmor.protection": "马儿保护",
+
+	"advancements.husbandry.netheritehorsearmor.obtain_netherite_horse_armor.title": "俺的爱马！",
```

原文：`True Love!`
翻译应以原文为准。请不要加私货
此类问题请全文通查并修改

```suggestion
	"advancements.husbandry.netheritehorsearmor.obtain_netherite_horse_armor.title": "真爱！",
```

---

好的